.semgrep.yml

```diff
@@ -9,9 +9,9 @@ rules:
       Copyright (c) $Y Aiven, Helsinki, Finland. https://aiven.io/
   - metavariable-regex:
       metavariable: $Y
-      regex: ^(?!2018-2021)
```

Comment:
I think we should completely remove this semgrep rule, since we do not use any copyright headers